python/lsst/summit/utils/astrometry/plotting.py

```diff
@@ -0,0 +1,133 @@
+# This file is part of summit_utils.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import numpy as np
+
+import matplotlib.pyplot as plt
+from astropy.coordinates import Angle
+import astropy.units as u
+
+from lsst.obs.lsst.translators.latiss import AUXTEL_LOCATION
+from .. import quickSmooth
+
+
+# TODO: Add some of Craig's nice overlay stuff here
+
+def plot(exp, icSrc=None, filteredSources=None, saveAs=None):
+    """Plot an exposure, overlaying the selected sources and compass arrows.
+
+    Plots the exposure on a logNorm scale, with the brightest sources, as
+    selected by the configuration, overlaid with an x. Plots compass arrows
+    for both north/east and az/el. Optionally saves the output to a file is
+    ``saveAs`` is supplied.
+
+    Parameters
+    ----------
+    exp : `lsst.afw.image.Exposure`
+        The exposure to get the astrometry for.
+    icSrc : `lsst.afw.table.SourceCatalog`
+        The source catalog for the exposure.
+    filteredSources : `lsst.afw.table.SourceCatalog`, optional
+        The filtered source catalog. If supplied, shows which sources were
+        selected.
+    saveAs : `str`, optional
+        Saves the plot to this filename if specified.
+    """
+    plt.figure(figsize=(16, 16))
+    arr = exp.image.array
+    arr = np.clip(arr, 1, 1000000)
```

Comment:
Should the upper limit be a kwarg?

Reply:
Arguably the lower limit should be perhaps, as that's the functional one. I can't see an image with values over 1e6 though. I'll do both though, why not.